metadata-ingestion/tests/unit/glue/glue_mces_golden.json

```diff
@@ -1,4 +1,43 @@
 [
+  {
```

Comment:
Have the test coverage numbers remained the same or dropped after this change?

Reply:
It remained the same==86%.